assimilation_code/modules/assimilation/probit_transform_mod.f90

```diff
@@ -141,8 +141,7 @@ subroutine transform_to_probit(ens_size, state_ens_in, distribution_type, p, &
    call to_probit_gamma(ens_size, state_ens, p, probit_ens, use_input_p, &
       bounded_below, bounded_above, lower_bound, upper_bound)
```

Comment:
Similar to `to_probit_beta` change below, since bounds are fixed [0,unbounded)
```suggestion
   call to_probit_gamma(ens_size, state_ens, p, probit_ens, use_input_p)
```

Reply:
also subroutine to_probit_gamma(ens_size, state_ens, p, probit_ens, use_input_p)

Reply:
fixed in bcbeefe23043c149dd6eb6f906194f9bc1a61f69